apps/test/unit/storage/FirebaseStorageTest.js

```diff
@@ -895,9 +897,11 @@ describe('FirebaseStorage', () => {
             .set(EXISTING_COUNTER_DATA)
         )
         .then(() => {
-          FirebaseStorage.populateTable(
+          let firebasePromise = FirebaseStorage.populateTable(
```

Comment:
Does this need to be a variable? Could you just `FirebaseStorage.populateTable(...).then(...)` (and below)

Reply:
Good point. Thanks!